basic/05_trajectory.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * Generate a trajectory and execute it.
+ *
+ * For more information, go to http://docs.hebi.us/tools.html#cpp-api
+ *
+ * This script assumes you can create a group with 1 module.
+ *
+ * HEBI Robotics
+ * September 2018
+ */
+
+#include "trajectory.hpp"
+#include "Eigen/Eigen"
+
+#include "lookup.hpp"
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "command.hpp"
+#include <math.h>
```

Comment:
`<cmath>`

Reply:
I could be wrong here, but this was done because of some MSVC issues with namespaces.  Can talk more about this later.